Sources/x10/xla_client/xrt_computation_client.cc

```diff
@@ -136,6 +136,10 @@ class TensorAllocator : public tensorflow::Allocator {
     void* ptr = _aligned_malloc(
         alloc_blocks->alloc_key.alignment,
         alloc_blocks->alloc_key.alignment + alloc_blocks->alloc_key.num_bytes);
+#elif defined(__APPLE__)
```

Comment:
It'd be better to create a helper for platform-dependent aligned allocation, I think. It becomes hard to read with all these preprocessor noise inlined.

Reply:
Sure, I can try to do that.  Would you mind if I did that in a follow up?

Reply:
No, sounds good.